db/migrate/20220816233917_add_user_ref_to_likes.rb

```diff
@@ -0,0 +1,7 @@
+# rubocop:disable all
+
+class AddUserRefToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
[ ] Same here for comments, Please fix this migration to add the reference between `liks` table and `users` table as `author_id` not `user_id`.

[hint] same goes here you can add this:
```ruby
add_reference :likes, :author, null: false, foreign_key: { to_table: :users }
```